imgparse/imgparse.py

```diff
@@ -550,3 +606,29 @@ def get_bandnames(image_path=None, exif_data=None, xmp_data=None):
     except KeyError:
         logger.error("Couldn't parse bandnames. Sensor might not be supported")
         raise ParsingError("Couldn't parse bandnames. Sensor might not be supported")
+
+
+@get_if_needed("exif_data", getter=get_exif_data, getter_args=["image_path"])
+@get_if_needed("xmp_data", getter=get_xmp_data, getter_args=["image_path"])
+def get_home_point(image_path, exif_data=None, xmp_data=None):
+    """
+    Get the flight home point. Used for `get_relative_altitude(alt_source=terrain)`.
+
+    :param image_path: the full path to the image
+    :param exif_data: used internally for memoization. Not necessary to supply.
+    :param xmp_data: used internally for memoization. Not necessary to supply.
+    :return: **lat**, **lon** - coordinates of flight home point
+    """
+    try:
+        make, model = get_make_and_model(image_path, exif_data)
+        if make == "DJI":
+            self_data = xmp_data[xmp.DJITags.SELF_DATA].split("|")
+            if len(self_data) == 4:
+                return self_data[0], self_data[1]
+            else:
+                raise KeyError()
+        else:
```

Comment:
I'm surprised this feature isn't supported on Sentera sensors. Am I missing something here?